src/Swarm/Language/Parse.hs

```diff
@@ -346,7 +348,16 @@ unOps = Map.unionsWith (++) $ mapMaybe unOpToTuple allConst
     pure $
       Map.singleton
         (fixity ci)
-        [assI (exprLoc1 $ SApp (noLoc $ TConst c) <$ symbol (syntax ci))]
+        [assI (exprLoc1 $ SApp (noLoc $ TConst c) <$ operatorString (syntax ci))]
+
+operatorString :: Text -> Parser Text
+operatorString n = (lexeme . try) (string n <* notFollowedBy operatorSymbol)
+
+operatorSymbol :: Parser Text
+operatorSymbol = T.singleton <$> oneOf opChars
+ where
+  isOp = \case { ConstMFunc {} -> False; _ -> True } . constMeta
+  opChars = concatMap (from . syntax) . filter isOp $ map constInfo allConst
```

Comment:
Should we call `nub` on `opChars`?  If I'm reading this correctly, it may end up with multiple copies of the same character for characters that occur in multiple operators (`=`, `<`, etc.).  I doubt it makes a huge difference but I suppose `oneOf` might have to do some redundant work if `opChars` has duplicates.